server/journeys/screen/screen/controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { BaseJourneyController } from '../../base/controller'
+import { sortAscending } from '../../../utils/utils'
+
+export class ScreenController extends BaseJourneyController {
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const outcomeTypeItems = this.sortRadios(
+      await this.getReferenceDataOptionsForRadios(
+        req,
+        'outcome-type',
+        res.locals.formResponses?.['outcomeType'] || req.journeyData.saferCustodyScreening?.outcomeType || '',
+      ),
+    )
+
+    res.render('screen/screen/view', {
+      outcomeTypeItems,
+      reasonForDecision:
+        res.locals.formResponses?.['reasonForDecision'] || req.journeyData.saferCustodyScreening?.reasonForDecision,
+    })
+  }
+
+  POST = async (req: Request<unknown, unknown, SchemaType>, res: Response): Promise<void> => {
+    req.journeyData.saferCustodyScreening!.outcomeType = req.body.outcomeType
+    req.journeyData.saferCustodyScreening!.reasonForDecision = req.body.reasonForDecision
+    res.redirect('check-answers')
+  }
+
+  private sortRadios = (items: { value: string; text: string | undefined }[]) => {
+    // TODO: No need to manually change ACC once ReferenceData DB is updated
```

Comment:
Much better not to do this content change here. There's a ticket to update it in the back end and it's the kind of thing that gets forgotten about